package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "lfg-omega13",
-  "version": "0.1.1",
-  "description": "A big red button that sends a very aggressive phone notification.",
+  "version": "0.1.2",
```

Comment:
You had this right before. The config format has changed, so this is indeed breaking and needs to be `0.2.0`.